resources/lang/ru/validation.php

```diff
@@ -33,40 +33,82 @@
     "boolean"              => "Поле :attribute должно иметь значение логического типа.", // калька 'истина' или 'ложь' звучала бы слишком неестественно
     "confirmed"            => "Поле :attribute не совпадает с подтверждением.",
     "date"                 => "Поле :attribute не является датой.",
+    'date_equals'          => 'Атрибут :attribute должен быть датой, равной :date.',
     "date_format"          => "Поле :attribute не соответствует формату :format.",
     "different"            => "Поля :attribute и :other должны различаться.",
     "digits"               => "Длина цифрового поля :attribute должна быть :digits.",
     "digits_between"       => "Длина цифрового поля :attribute должна быть между :min и :max.",
+    'dimensions'           => ':attribute имеет недопустимые размеры изображения.
+    .',
```

Comment:
I guess here there is a new line that should not be present

Reply:
ok